// containers/calc1/index.js
import { connect } from 'react-redux'
import React, { Component } from 'react'
import { Link } from 'react-router'
import Slider from 'react-rangeslider'
import 'react-rangeslider/lib/index.css'
import Menu from '../menu'
import { ToastContainer, toast } from 'react-toastify'
import 'react-toastify/dist/ReactToastify.css'


class Calc_1 extends Component {

    constructor(props, context) {
        super(props, context)
        this.state = {
            volume: 0
        }
    }
    componentDidMount() {
        window.scrollTo(0, 0)
    }
    handleOnChange = (value) => {
        this.setState({
            volume: value
        })
    }
    notify = () => toast.error(`Вы не можете продолжить так как ваш баланс равен - 0`, {
        position: toast.POSITION.BOTTOM_RIGHT
    })
    render() {
        let { volume } = this.state
        const willHave = this.state.volume * 3.2
        const horizontalLabels = {
            0: 'Low',
            300: 'Popular',
            650: 'High'
        }
         
        return (
            <div>
                <div>
                    <div className="calc-header">
                        <div className="header-overlay">
                            <Menu />
                            <div className="price-header-content">
                                <h2 className="price-header-title">Shoose best price plan </h2>
                                <p className="price-header-text">This is standart plan - 10% daily profit</p>
                            </div>{/* header-content */}
                        </div>{/* header-overlay */}
                    </div>{/* calc-header */}
                    <div className="calc-container">
                        <div className="wrapper">
                            <div className="calc-item">
                                <img src="/img/plant.svg" alt="notes" />
                                <span className="calc-price">&nbsp; &nbsp;&nbsp;Invest: {this.state.volume}$</span>
                            </div>{/* calc-item */}
                            <div className="calc-item">
                                <img src="/img/growth.svg" alt="money" />
                                <span className="calc-price">&nbsp;&nbsp;&nbsp;Profit : {willHave.toFixed(1)}$</span>
                            </div>{/* calc-item */}
                            <div className="calc-scroll-container">
                                <Slider
                                    min={50}
                                    max={650}
                                    value={volume}
                                    labels={horizontalLabels}
                                    orientation="horizontal"
                                    onChange={this.handleOnChange}
                                />
                            </div>{/* calc-scroll-container */}
                            <button className="calc-item-btn" onClick={this.notify}>sign in</button>

                            {(!this.props.hideButton) ? null : <ToastContainer />}

                        </div>{/* wrapper */}
                    </div>{/* calc-container */}
                    <div className="footer">
                        <div className="wrapper">
                            <div className="footer-container">
                                <div className="footer-menu">
                                    <span className="footer-menu-title">Help &amp; Support</span>
                                    <button className="footer-menu-btn">read more</button>
                                </div>{/* footer-menu */}
                                <div className="footer-menu">
                                    <span className="footer-menu-title">FAQ</span>
                                    <button className="footer-menu-btn">read more</button>
                                </div>{/* footer-menu */}
                            </div>{/* footer-container */}
                            <p className="footer-text">Packages for mobile, IoT, front end, back end, robotics… everything you need to start building amazing things.</p>
                        </div>{/* wrapper */}
                        <p className="footer-copyright">© 2012 – 2018, www.sitename.by</p>
                    </div>{/* footer */}
                </div>

            </div>
        )
    }
}
const mapStateToProps = state => ({
    hideButton: state.auth.hideButton
})
export default connect(mapStateToProps, null)(Calc_1)
