// api/mockNews.js
export default [
    {
        id: "1",
        categoryId: "1",
        name: "Robo_trade - real profit in 2018",
        description: "Реальный доход компании от доли в торговом боте состовляет 1,437,823,3$ за первый квартал , так как наша команда удерживает 40% всей доли в торговом боте ",
        price: 823,
        image: "/img/index-back-1.jpg",
    },
    {
        id: "2",
        categoryId: "1",
        name: "10.04 мы листим trx, в этот день торговый бот даст 300-400%",
        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
        price: 953,
        image: "/img/index-back-2.jpg",

    },
    {
        id: "3",
        categoryId: "1",
        name: "Огромный куш",
        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
        price: 953,
        image: "/img/index-back-3.jpg",

    }, {
        id: "4",
        categoryId: "1",
        name: "Седьмой инвестор стал доллоровым миллионером ",
        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
        price: 953,
        image: "/img/index-back-4.jpeg",

    }, {
        id: "5",
        categoryId: "1",
        name: "Как долго работает компания?",
        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
        price: 953,
        image: "/img/index-back-5.jpg",

    },
    {
        id: "6",
        categoryId: "1",
        name: "Мы переезжаем в Тайвань или 'подальше от закона '",
        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
        price: 953,
        image: "/img/btc.png",

    },]
